packages/shared/lib/core/network/utils/getEnabledNetworkFromFeatureFlags.ts

```diff
@@ -2,10 +2,19 @@ import features from '@features/features'
 import { NetworkId } from '../enums'
 
 export function getEnabledNetworkFromFeatureFlags(): NetworkId {
-    if (features.onboarding[NetworkId.Iota]?.enabled || features.onboarding[NetworkId.IotaAlphanet]?.enabled) {
-        return features.onboarding[NetworkId.Iota]?.enabled ? NetworkId.Iota : NetworkId.IotaAlphanet
-    } else if (features.onboarding[NetworkId.Shimmer]?.enabled || features.onboarding[NetworkId.Testnet]?.enabled) {
-        return features.onboarding[NetworkId.Shimmer]?.enabled ? NetworkId.Shimmer : NetworkId.Testnet
+    if (
+        features.onboarding[NetworkId.Iota]?.enabled ||
+        features.onboarding[NetworkId.IotaTestnet]?.enabled ||
+        features.onboarding[NetworkId.IotaAlphanet]?.enabled
+    ) {
+        return features.onboarding[NetworkId.Iota]?.enabled || features.onboarding[NetworkId.IotaTestnet]?.enabled
+            ? NetworkId.Iota
+            : NetworkId.IotaAlphanet
```

Comment:
I think this logic is not correct because it is never returned `IotaTestnet`

Reply:
yeah this was a ghetto patch xD because I didnt know what to add, this is just to predict which app it is based on the networks enabled to display a correct app name, i'll update :trollface:

Reply:
cool :trollface: